lib/pbench/cli/agent/commands/tools/list.py

```diff
@@ -0,0 +1,142 @@
+"""
+pbench-list-tools
+
+This script can list all tools from all groups, list tools from a specific
+group, or list which gourps contain a specific tool
+
+"""
+
+import logging
+import sys
+
+import click
+
+from pbench.agent.utils import setup_logging
+from pbench.cli.agent import CliContext, pass_cli_context
+from pbench.cli.agent.commands.tools.base import ToolCommand
+from pbench.cli.agent.options import common_options
+
+LOG = logging.getLogger(__name__)
+
+
+class ListTools(ToolCommand):
+    """ List registered Tools """
+
+    def __init__(self, context):
+        super(ListTools, self).__init__(context)
+
+        setup_logging(debug=False, logfile=self.pbench_log)
```

Comment:
I am not sure why I missed this in the past, but this method, while it might work per tool, has potential to cause conflicts if used by other components.  Let's move to a logging setup like we have on the server side.

Reply:
I addressed this in https://github.com/distributed-system-analysis/pbench/pull/1926